instrumentation/elastic-opentelemetry-instrumentation-openai/pyproject.toml

```diff
@@ -18,18 +18,17 @@ classifiers = [
     "License :: OSI Approved :: Apache Software License",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3.13",
 ]
 dependencies = [
-    # 1.31.0 is required for proper histogram bucket advisory support
-    "opentelemetry-api ~= 1.31",
-    "opentelemetry-instrumentation ~= 0.52b0",
-    "opentelemetry-semantic-conventions ~= 0.52b0",
+    # 1.35.0 is required for LogRecord event_name parameter
+    "opentelemetry-api ~= 1.35.0",
```

Comment:
```suggestion
    "opentelemetry-api ~= 1.35",
```

Otherwise it is `>=1.35.0, < 1.36`

Reply:
Sorry for directly committing the suggestion, wanted to verify e2e. Could confirm it works

https://github.com/elastic/observability-examples/pull/78/commits/72018bdbcff0d331667217b90ab0712aa74316a0